WTWSMS/history/titles/c_calatrava.txt

```diff
@@ -12,10 +12,6 @@
 531.1.1={ 
 	holder=190069 #�iudeis
 }
-537.1.1={
-	holder=36555 # Athanagild Athanagilding
-	liege = "k_visigoths"
-}
```

Comment:
Isn't this going to break history in this province?

Reply:
I'm not sure, although I think not. I'll check later.

Reply:
Ok, tell me when you have seen.

Reply:
Damn, forgot to answer this. I´m sorry it was solved in a commit some time after your comment.

Reply:
The history of the province or it not breaking?

Reply:
It not breaking

Reply:
It not breaking

Reply:
Who holds it then? I don't understand the purpose here.

Reply:
The Visigoths until 567 when Orospeda takes it.

Reply:
Why shouldn't Athanagild hold it?

Reply:
I don´t actually remember. It is held by the King.

Reply:
In the master Athanagild holds Orospeda and Calatrava beneath the king.

Reply:
Oh yeah, I did that to have Athanagild simply holding Seville

Reply:
Why?

Reply:
IDK, it seemed as a good idea to me at the moment but that's a very small change so I can return it back if you want

Reply:
I just wanted to understand how you thought, it isn't necessarily bad.

Reply:
Then let's keep it there if you don't mind

Reply:
The thing is that I don´t actually know why I changed it, I suppose to have Athanagild´s rebellion´s land smaller.

Reply:
That could however mean the rebellion is less dangerous. And if he held this land historically it is important to represent that.

Reply:
This is still relevant.